packages/ui/react-appkit/src/hooks/useTelemetry.ts

```diff
@@ -24,32 +21,28 @@ export type UseTelemetryOptions = {
  *
  * Must be called inside a react router context.
  */
-export const useTelemetry = ({ namespace, router = true }: UseTelemetryOptions) => {
+export const useTelemetry = ({ namespace, observability, router = true }: UseTelemetryOptions) => {
```

Comment:
FYI this whole package is deprecated and we're trying to delete it so don't worry about updating it fully